client/vnctl/lib/vnctl/cli/datapath.rb

```diff
@@ -10,6 +10,7 @@ class Datapath < Base
       option :is_connected, :type => :boolean, :desc => "Flag that detemines if the datapath is connected or not."
       option :dpid, :type => :string, :desc => "Hexadecimal id for the datapath."
       option :node_id, :type => :string, :desc => "The node id for the datapath."
+      option :enable_ovs_learn_action, :type => :boolean, :desc => "Enables the datapath to use OVS learning flows."
```

Comment:
The default value is in the database, and the webapi shouldn't imo pass down values to node_api that the user didn't explicitly supply. It means we'll have the default value defined in two different places.